src/main/resources/db/migration/V2024_06_21_1166__CIV-14169-update-defendant-task-list-final-orders-issued.sql

```diff
@@ -2,7 +2,7 @@
  * Add scenario
  */
```

Comment:
The production database scripts will have been run by the time the code freeze is lifted. All changes to previous scripts need to be done as update SQLs now.